Objects/bytes_methods.c

```diff
@@ -557,10 +557,52 @@ find_internal(const char *str, Py_ssize_t len,
 }
 
 PyObject *
-_Py_bytes_find(const char *str, Py_ssize_t len, PyObject *sub,
+_Py_bytes_find(const char *str, Py_ssize_t len, PyObject *subobj,
                Py_ssize_t start, Py_ssize_t end)
 {
-    Py_ssize_t result = find_internal(str, len, "find", sub, start, end, +1);
+    Py_ssize_t result;
+    if (PyTuple_Check(subobj)) {
+        result = -1;
+        ADJUST_INDICES(start, end, len);
+        // Work in batches of 10000
+        for (; result == -1 && start <= end; start += 10000) {
```

Comment:
Should we define this as a constant somewhere? In that case, where?

Reply:
```suggestion
        For now I think a simple `const` at the top of the function is ok. It is possible (although not very likely) that different constants for different methods will be needed in the future. Until it is known for certain, best to keep it flexible I think.
```

Reply:
A macro constant is fine too, it can always be duplicated later.